### ISSUE TITLE
Scroll without specifying element

### ISSUE THREAD

**Issue by chenchaoyi** [appium/appium-xcuitest-driver#88]
With the current Appium and Node.js client WD, it accepts
```
driver.execute("mobile: scroll", [{ direction: 'up' }])
```
to do the scroll operation.

With XCUItest driver, it seems an extra `element` is required now:
```
[HTTP] --> POST /wd/hub/session/34ac6c12-eb9a-466a-a4cf-8c67086bf189/execute {"script":"mobile: scroll","args":[{"direction":"down"}]}
[MJSONWP] Calling AppiumDriver.execute() with args: ["mobile: scroll",[{"direct...
[debug] [MJSONWP] Bad parameters: BadParametersError: Parameters were incorrect. We wanted "Mobile scroll needs an element" and you sent undefined
[HTTP] <-- POST /wd/hub/session/34ac6c12-eb9a-466a-a4cf-8c67086bf189/execute 400 5 ms - 92
```

And it seems @yaphatak 's [PR](https://github.com/facebook/WebDriverAgent/pull/231) for WDA was not accepted.

I am wondering what's the equivalent of scrolling the entire UIApplication without specifying an element? And if we have to add element now, is there anyway to do it in the chainable API way in WD.js ?

**Comment by yaphatak** [comment]
i submitted pr here https://github.com/appium/appium-xcuitest-driver/pull/90

**Comment by imurchie** [comment]
This should be fixed by #90.
